scripts/next_version.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+
+set -eu
+
+# Update these as needed.
+MAJOR=0
+MINOR=13
```

Comment:
This isn't being used yet but once we can agree on a version scheme I'd start moving everything to be based on this.